SIDFactoryII/source/runtime/editor/editor_facility.cpp

```diff
@@ -100,9 +100,10 @@ namespace Editor
 		SIDConfiguration sid_configuration; // Default settings are applicable
 
 		const bool sid_use_resample = GetSingleConfigurationValue<ConfigValueInt>(config, "Sound.Emulation.Resample", 1) != 0;
+		const int sid_sample_frequency = GetSingleConfigurationValue<ConfigValueInt>(config, "Sound.Emulation.SampleFrequency", 44100);
```

Comment:
Should we safeguard against insane values? For instance make sure it is in a certain range, and use the default otherwise (and log a warning)?

Reply:
I added 11025Hz (most sensible lowest rate with reSID set to resampling mode, see source) to 192000Hz (what today's pro audio devices usually can do) as limits, plus logging.

Note that there's no real upper bound for reSID except the C64 clock rate, but I don't think more than 192k have ever been tested :)

Reply:
Thanks!